SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -20,9 +20,15 @@
 
 DEFINE_LOG_CATEGORY(LogSpatialPackageMap);
 
-void USpatialPackageMapClient::Init(USpatialNetDriver* InNetDriver)
+void USpatialPackageMapClient::Init(USpatialNetDriver* InNetDriver, FTimerManager* TimerManager)
```

Comment:
uber nit: InNetDriver should be renamed to NetDriver

Reply:
Fixed.